### ISSUE TITLE
BUILD EXPERIMENTAL 18w51a: Hotfixes and bricks shapes

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Skywanderers#1048]
I will now post build info on github, it might be more convenient. 

Mainly hotfixes: https://github.com/tsunamayo/Skywanderers/issues?q=is%3Aissue+is%3Aclosed

[CHANGES]

- tetra, hepta and corner shape for bricks.
- Debug menu: press F3 for a list of option. WIP, mainly just FPS for now.

Thanks as always for testing the game!

**Comment by LeMinaw** [comment]
Just a suggestion: maybe you should commit build infos in a markdown file of the repo, instead of using the issue tracker? It might allow us to keep track of the changes, and using webhooks, we should be able to automatically broadcast any changes to the discord server.

**Comment by tsunamayo** [comment]
Yes looks like an amazing idea, what does it consists off exactly? cheers

**Comment by LeMinaw** [comment]
You just have to create a file (eg. changelog.md) at the root of the repo, and edit (commit) it each time you release a new version (preferably latest version at top, older at bottom). You're allowed to use markdown to make it easier to read (lists, headers) just as there, GitHub will display markdown files nicely.

For pushing changes to discord, we have to setup a webhook so each time a file gets edited (committed) a little message is sent to discord. Several tutorials are available and this is quite straightforward, but I can do it if you want (just give me access to this repo).
